src/presence_analyzer/templates/mean_time_weekday.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'main.html' %}
+
+{% block script %}
+    <script type="text/javascript" src="/static/js/mean_time_weekday.js"></script>
+{% endblock %}
+
+{% block links %}
+    <ul>
+        <li><a href={{ url_for('presence_weekday') }}>Presence by weekday</a></li>
+        <li id="selected"><a href={{ url_for('mean_time_weekday') }}>Presence mean time</a></li>
+        <li><a href={{ url_for('presence_start_end') }}>Presence start-end</a></li>
+    </ul>
+{% endblock %}
+
+{% block h2 %}<h2>Presence mean time by weekday</h2>{% endblock %}
```

Comment:
`<h2>` can be moved to base and this block will be inside of this h2 in main. And please give this block a meaningful name since this is a title, h2 is way too enigmatic to name any block.